pages/configuration/https.rst

```diff
@@ -12,12 +12,25 @@ You also need to make sure that you have proper certificates in place, which are
 
 .. note:: If you're operating a single-node setup and would like to use HTTPS for the Graylog web interface and the Graylog REST API, it's possible to use :ref:`NGINX or Apache as a reverse proxy <configuring_webif_nginx>`.
 
+Thinks to consider
+==================
+
+You have multiple options to ensure that your connection is secure and save. The first would be to create a self-signed certificate, add that to the previous  copied java keystore and use this keystore with your Graylog java options.
```

Comment:
typo - should be "...your connection is secure and safe."
AND 
typo - should be "...add that to the previously"